extension/scripts/features/travelitemprofits/ttTravelItemProfits.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Travel Item Profits",
+		"travel",
+		() => settings.pages.travel.travelProfits,
+		null,
+		addProfitsColumn,
+		removeProfitsColumn,
+		{
+			storage: ["settings.pages.travel.travelProfits"],
+		},
+		async () => {
+			if (!hasAPIData()) return "No API access.";
+			await requireElement(".travel-agency-market .items-list-title");
```

Comment:
As told in the Discord before, those should go back in the execute function (`addProfitsColumn`).